phpBB/styles/prosilver/theme/common.css

```diff
@@ -603,6 +603,93 @@ a.header-avatar span:after {
 	display: none !important;
 }
 
+/* Smiley Dropdown---------------------------------------- */
+
+.dropdown .smiley-content {
+	border: 1px solid transparent;
+	border-radius: 5px;
+	position: relative;
+	z-index: 2;
+	display: inline-block;
+	overflow: hidden;
+	overflow-y: auto;
+	width: 200px;
+	max-height: 300px;
+	padding: 5px;
+}
+
```

Comment:
again why is this necessary just use the normal dropdown? if there is a slight difference then extend do not replace